lib/ansible/plugins/action/win_reboot.py

```diff
@@ -40,6 +40,17 @@ class ActionModule(RebootActionModule, ActionBase):
         'shutdown_timeout_sec': '2.5',
     }
 
+    BOOT_TIME_COMMANDS = {
+        'windows': DEFAULT_BOOT_TIME_COMMAND
+    }
+
+    def _get_platform(self):
+        return 'windows'
+
+    def _get_distribution(self):
+        version = self._low_level_execute_command('[System.Environment]::OSVersion.Version.Major')['stdout_lines'][0]
```

Comment:
This method is technically deprecated from Microsoft, if the executable manifest hasn't been set correctly it will always return `6.2` (Windows 8) even when it is newer. I don't see a need for this to be done in Windows and can potentially return None or a blank value as we shouldn't care there.

Reply:
I think just use a dummy value, if Windows does introduce a different method in the future we can deal with it then